### ISSUE TITLE
unable to build master

### ISSUE THREAD

**Issue by psteinb** [clij/clij#11]
just tried to build master bc87bd10059cc5c09f6b74e0e9ae0516a8fbb462 with no luck:

```
$ mvn clean install
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building CLIJ 0.6.0
[INFO] ------------------------------------------------------------------------
[WARNING] The POM for net.clearcontrol:clearcl:jar:0.6.7-m is missing, no dependency information available
Downloading: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml
[WARNING] Could not transfer metadata com.github.jnr:jffi:1.2.16-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml.
[WARNING] Failure to transfer com.github.jnr:jffi:1.2.16-SNAPSHOT/maven-metadata.xml from https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases was cached in the local repository, resolution will not be reattempted until the update interval of unidata has elapsed or updates are forced. Original error: Could not transfer metadata com.github.jnr:jffi:1.2.16-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml.
Downloading: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml
[WARNING] Could not transfer metadata com.github.jnr:jffi:1.3.0-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml.
[WARNING] Failure to transfer com.github.jnr:jffi:1.3.0-SNAPSHOT/maven-metadata.xml from https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases was cached in the local repository, resolution will not be reattempted until the update interval of unidata has elapsed or updates are forced. Original error: Could not transfer metadata com.github.jnr:jffi:1.3.0-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml.
[WARNING] Failure to transfer com.github.jnr:jffi:1.2.16-SNAPSHOT/maven-metadata.xml from https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases was cached in the local repository, resolution will not be reattempted until the update interval of unidata has elapsed or updates are forced. Original error: Could not transfer metadata com.github.jnr:jffi:1.2.16-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.2.16-SNAPSHOT/maven-metadata.xml.
[WARNING] Failure to transfer com.github.jnr:jffi:1.3.0-SNAPSHOT/maven-metadata.xml from https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases was cached in the local repository, resolution will not be reattempted until the update interval of unidata has elapsed or updates are forced. Original error: Could not transfer metadata com.github.jnr:jffi:1.3.0-SNAPSHOT/maven-metadata.xml from/to unidata (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases): Failed to transfer file: https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases/com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml. Return code is: 400 , ReasonPhrase:Repository version policy: RELEASE does not allow metadata in path: com/github/jnr/jffi/1.3.0-SNAPSHOT/maven-metadata.xml.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.451 s
[INFO] Finished at: 2019-01-04T10:48:15+01:00
[INFO] Final Memory: 28M/311M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project clij_: Could not resolve dependencies for project net.haesleinhuepf:clij_:jar:0.6.0: Failure to find net.clearcontrol:clearcl:jar:0.6.7-m in http://dl.bintray.com/haesleinhuepf/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of hslh has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

@haesleinhuepf I checked your bintray repo for this tool and indeed didn't find `clearcl 0.6.7-m`, I just found this 
![image](https://user-images.githubusercontent.com/1465603/50682330-ca3b9600-100e-11e9-94b8-a07da1460cac.png)

**Comment by haesleinhuepf** [comment]
Hey @psteinb ,

Happy New Year!

Current version of clij is 0.10.5 could you please try to git pull? Furthermore, the git repositories URL changed since last year. It's now 
https://github.com/clij/clij.git

Does that help?

Cheers,
Robert

**Comment by psteinb** [comment]
Happy New Year! I did a fresh clone on a nother machine (using openjdk 1.8.0_172 IIRC).
There, I see the following on master:
```
$ mvn clean install
[INFO] Scanning for projects...
Downloading from clij: http://dl.bintray.com/haesleinhuepf/clij/net/haesleinhuepf/clij-parent-pom/0.3.3/clij-parent-pom-0.3.3.pom
Downloaded from clij: http://dl.bintray.com/haesleinhuepf/clij/net/haesleinhuepf/clij-parent-pom/0.3.3/clij-parent-pom-0.3.3.pom (7.3 kB at 7.9 kB/s)
...
[INFO] 
[INFO] ----------------------< net.haesleinhuepf:clij_ >-----------------------
[INFO] Building clij 0.10.4
[INFO] --------------------------------[ jar ]---------------------------------
...
Downloaded from central: https://repo.maven.apache.org/maven2/com/thoughtworks/qdox/qdox/2.0-M7/qdox-2.0-M7.jar (315 kB at 3.0 MB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-compiler-javac/2.8.2/plexus-compiler-javac-2.8.2.jar (20 kB at 193 kB/s)
[INFO] Compiling 134 source files to /projects/steinbac/development/clij/target/classes
[INFO] /projects/steinbac/development/clij/src/main/java/net/haesleinhuepf/clij/CLIJ.java: Some input files use unchecked or unsafe operations.
[INFO] /projects/steinbac/development/clij/src/main/java/net/haesleinhuepf/clij/CLIJ.java: Recompile with -Xlint:unchecked for details.
[INFO] -------------------------------------------------------------
[WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] /projects/steinbac/development/clij/src/main/java/net/haesleinhuepf/clij/utilities/CLKernelExecutor.java:[7,51] com.sun.org.apache.xalan.internal.xsltc.dom.SimpleResultTreeImpl is internal proprietary API and may be removed in a future release
[INFO] 1 warning
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /projects/steinbac/development/clij/src/main/java/net/haesleinhuepf/clij/macro/modules/CLInfo.java:[18,8] class ClInfo is public, should be declared in a file named ClInfo.java
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11:56 min
[INFO] Finished at: 2019-01-04T11:18:15+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:compile (default-compile) on project clij_: Compilation failure
[ERROR] /projects/steinbac/development/clij/src/main/java/net/haesleinhuepf/clij/macro/modules/CLInfo.java:[18,8] class ClInfo is public, should be declared in a file named ClInfo.java
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

It looks like this is the current state of affairs. Not sure, why maven reports `0.10.4` instead of the head tag `0.10.5`.

**Comment by psteinb** [comment]
NB the error above remains when I checkout `0.10.4` explicitely. I'll prepare a PR to try to fix it.

**Comment by psteinb** [comment]
looks good to me now. I fixed the above mentioned problem with #12 
btw, the default tests open a ton of GUI windows on my machine. this may not be optimal and I guess is the reason why clij-benchmarks exists.
